### DISCUSSION TITLE
How to remove default canonical tag?

### DISCUSSION THREAD

**Discussion by wojtazik** [garmeeh/next-seo#1072] [Q&A]
I mean, when i don't pass the canonical tag to the NextSeo component, the `<link rel="canonical" href="https://">` will be rendered. I need to render nothing if the canonical tag is set to falsy value or is not set.

**Comment by garmeeh** [comment]
Are you using `<DefaultSeo />`? Maybe you are setting one in there? 

If so you will need to remove from there.
